src/main/primitives/Matrix3x3.java

```diff
@@ -0,0 +1,170 @@
+package primitives;
+
+import util.NormalCompare;
+
+/**
+ * The {@link Vector} class represents a {@link Vector} with it's base at the origin and it's head at the {@link Point}
+ * 'head'.
+ *
+ * @author Abraham Murciano
+ * @author Eli Levin
+ */
+public class Matrix3x3 {
+
+	// rows of matrix
+	public Vector r1;
+	public Vector r2;
+	public Vector r3;
+	public static final Matrix3x3 IDENTITY =
+			new Matrix3x3(new Vector(1, 0, 0), new Vector(0, 1, 0), new Vector(0, 0, 1));
+
+	public Matrix3x3(Vector r1, Vector r2, Vector r3) {
+		this.r1 = r1;
+		this.r2 = r2;
+		this.r3 = r3;
+	}
+
+	public Matrix3x3(Matrix3x3 A) {
+		this(A.r1, A.r2, A.r3);
+	}
```

Comment:
Your matrix class seems immutable (assuming your vecors are intended to be final). therefore no need for a copy ctor. you can return the same instance without having to worry about other ppl changing it